src/main/java/uk/gov/companieshouse/confirmationstatementapi/eligibility/impl/CompanyOfficerValidation.java

```diff
@@ -46,8 +46,8 @@ public void validate(CompanyProfileApi companyProfileApi) throws EligibilityExce
                 throw new EligibilityException(EligibilityStatusCode.INVALID_COMPANY_APPOINTMENTS_INVALID_NUMBER_OF_OFFICERS);
             }
         } else {
-            if(activeOfficersCount != null && activeOfficersCount > 5) {
-                ApiLogger.info(String.format("Company Officers validation failed for: %s. More than 5 active officers.", companyProfileApi.getCompanyNumber()));
+            if(activeOfficersCount != null && (activeOfficersCount == 0 || activeOfficersCount > 5)) {
+                ApiLogger.info(String.format("Company Officers validation failed for: %s. No officers found or more than 5 active officers.", companyProfileApi.getCompanyNumber()));
```

Comment:
I would add the activecount into the log message as well, have it after the  message like this "...No officers found or more than 5 active officers, activeOfficersCount = %s